docs/types.rst

```diff
@@ -14,90 +14,279 @@ to form complex types.
 In addition, types can interact with each other in expressions containing
 operators.
 
+***********
 Value Types
-===========
+***********
 
 The following types are also called value types because variables of these
 types will always be passed by value, i.e. they are always copied when they
 are used as function arguments or in assignments.
 
 .. index:: ! bool, ! true, ! false
 
-Booleans
---------
+Boolean
+=======
+**Keyword:** ``bool``
 
-``bool``: The possible values are constants ``true`` and ``false``.
-
-Operators:
-
-*  ``not`` (logical negation)
-*  ``and`` (logical conjunction, "&&")
-*  ``or`` (logical disjunction, "||")
-*  ``==`` (equality)
-*  ``not ... == ... ``, ``!=`` (inequality)
-
-The operators ``or`` and ``and`` apply the common short-circuiting rules. This means that in the expression ``f(x) or g(y)``, if ``f(x)`` evaluates to ``true``, ``g(y)`` will not be evaluated even if it may have side-effects.
-
-.. index:: ! uint, ! int, ! integer
+A Boolean is a type to store a logical/truth value.
 
+Values
+------
+The only possible values are the constants ``true`` and ``false``.
+
+Operators
+---------
+
+====================  ===================  
+Operator              Description
+====================  ===================  
+``f(x) not g(y)``     Logical negation     
```

Comment:
Please use `x not y` instead of `f(x)` notation. It is more confusing, and we're not describing functional flows or anything like that.

Reply:
@fubuloubu Will change with the next push when I finish the rest! Was lazy and didn't change that from the previous version :P 
You mentioned "some issues": Apart from this issue (and the weird currency1 stuff) is there anything else or just some proof reading? If it's the latter: I will do that when the rest is finished.
Thanks for the feedback!